frontend/src/app/payment/edit/page.tsx

```diff
@@ -171,7 +243,64 @@ export default function PaymentPage() {
             Upload new receipt
           </div>
           <div className="mx-10 mb-20">
-            <UploadSlip />
+            <div className="">
+              <div className="">
+                {imagePreview ? (
+                  <div className="flex items-center justify-center">
+                    <Image
+                      src={imagePreview}
```

Comment:
LGTM, 

The reason it didn't work recently is that I forgot to include the useState() return to the parent component.
My apologies for the oversight.

Reply:
Oh its ok, ค่อยแก้รอบหน้าที่มีโอกาส5555555